parsers/svg-to-jsx/README.md

```diff
@@ -0,0 +1,203 @@
+| `formatConfig.endOfLine` | optional     | `auto` `lf` `crlf` `cr`                                     | `auto`           | [Prettier documentation](https://prettier.io/docs/en/options.html#end-of-line) |
+| `formatConfig.tabWidth`  | optional     | `number`                                                 | `2`           | [Prettier documentation](https://prettier.io/docs/en/options.html#tab-width)   |
+| `formatConfig.useTabs`   | optional     | `boolean`                                                | `true`           | [Prettier documentation](https://prettier.io/docs/en/options.html#tabs)        |
+| `formatConfig.singleQuote`   | optional     | `boolean`                                                | `false`           | [Prettier documentation](https://prettier.io/docs/en/options.html#quotes)        |
+| `formatConfig.exportDefault`   | optional     | `boolean`                                                | `true`           | Defines if the export of the jsx component should be done using default or not      |
+| `formatConfig.variableFormat`             | optional     | `camelCase` `kebabCase` `snakeCase` `pascalCase`      | `kebabCase` | The case transformation you want to apply to the name of js variable will be exported            |
+| `wrapper.tag`  | optional     | `string`          |  | An html tag will be used to encapsulate the svg tag              |
+| `wrapper.className`  | optional     | `string`    |      | A string or a pattern used to set a className attribute on the tag that encapsulate the svg tag                           |
+
+
+## Types
+
+ℹ️ **Please be aware that, depending on the order you use parsers, their input and output types have to match.**
+
+### Input
+
+Array of object with at least the key `value.url` or `value.content`
+
+```ts
+Array<
+  Record<string, any> & {
+    type: string;
+    name: string;
+    value: ({ url?: string } | { content?: string }) & { [key: string]: any };
+  }
+>
+```
+### Output
+
+Input type extended with `value.content` and `value.fileName`
+
+```ts
+Array<
+  Record<string, any> & {
+    type: string; 
+    value: { content: string; fileName: string; [key: string]: any }
+  }
+>;
+```
+
+
+## Basic usage
+### Config
+
+```json
+{
+  "name": "svg-to-jsx",
+  "options": {
+    "formatConfig": {
+      "exportDefault": true
+    }
+  }
+}
+```
+
+### Before/After
+#### Input
+
+```json5
+{
+  "type": "vector",
+  "name": "activity.svg",
+  "value": {
+    "url": "https://s3-us-west-2.amazonaws.com/figma-alpha-api/img/99b5/b311/257c650341b701d691be78f247b9cf5e"
+    //...
+  }
+  //...
+}
+```
+
+#### Output
+
+ℹ️  The `value.content` is the content of the file. It will be written as a file if the path of the rule in your config file is a directory.
+Otherwise, it's a simple json object written in the file defined as a path.
+
+```
+{
+  "type": "vector",
+  "name": "activity",
+  "value": {
+    "fileName": "Activity.jsx",
```

Comment:
I find it weird to call this prop `fileName` if we also include the file extension. Why not just call it `file`?

Reply:
It's not specific to this parser. It's used by the CLI and other destinations providers.
It's out of context.